Cargo.toml

```diff
@@ -10,5 +10,5 @@ version = "0.1.0"
 
 [dependencies]
 chrono = "0.4.19"
-getopts = "0.2.21"
+clap = { version = "3.2.5", features = ["derive"] }
```

Comment:
clap = { version = "3.2.6", features = [ "derive" ] }

Reply:
We should rather think about setting up the depenabot, should we not?

Reply:
I do not think we need to update clap that frequently. (Or probably at all tbh) 😅